coc.md

```diff
@@ -1,6 +1,32 @@
 # Code of Conduct
-We follow a [Code of Conduct for PyConPH](https://github.com/pythonph/pycon-2015/blob/master/coc.rst) and make sure all attendees and volunteers abide by it.
+The attendees and volunteers are required to abide and follow the [Code of Conduct for PyCon PH](https://pycon.python.ph/code-of-conduct/).
 
-We also normally ask volunteers to read the [Python Software Foundation's Code of Conduct](https://www.python.org/psf/codeofconduct/) and make sure they understand, [sign](https://docs.google.com/forms/d/1mky-a8ZQ8b9kLvE0bwURETJWHX0opO2tTBleRzcdW3g/viewform) and agree to it before they proceed.
+In addition, volunteers are required to read the [Python Software Foundation's Code of Conduct](https://www.python.org/psf/codeofconduct/), [sign](https://docs.google.com/forms/d/1mky-a8ZQ8b9kLvE0bwURETJWHX0opO2tTBleRzcdW3g/viewform) and agree to it before they are accepted as a volunteer.
 
-This is so we can plan the conference around the same ideals and be able to enforce a code of conduct after having seen that the PyConPH core team (committee heads, volunteers) themselves follow it. 
+## Rationale
+
+PyCon is a community conference where attendees are from different parts of life. To accomodate this diversity, the organizers must prepare an environment where participants will feel safe and respected, regardless of their gender, personal belief, or opinions. This is to promote colaboration and a spirit of community.
```

Comment:
"parts of life" -> "walks of life", I think.

Also, "collaboration" is misspelled in the last sentence.